README.md

```diff
@@ -198,9 +198,9 @@ $ puma -b 'ssl://127.0.0.1:9292?key=path_to_key&cert=path_to_cert'
 ```
 #### Self-signed SSL certificates (via the [`localhost`] gem, for development use):
 
-Puma supports the [`localhost`] gem for self-signed certificates. This is particularly useful if you want to use Puma with SSL locally, and self-signed certificates will work for your use-case. Currently, the integration can only be used in MRI.
+Puma supports the [`localhost`] gem for self-signed certificates. This is particularly useful if you want to use Puma with SSL locally, and self-signed certificates will work for your use-case. Currently, the integration can only be used in MRI and you need to require the [`localhost`] gem in your Puma configuration file and not in your rackup file [`config.ru`].
```

Comment:
I don't think we need to include what to not do in the docs.

```suggestion
Puma supports the [`localhost`] gem for self-signed certificates. This is particularly useful if you want to use Puma with SSL locally, and self-signed certificates will work for your use-case. Currently, the integration can only be used in MRI and you need to require the [`localhost`] gem in your Puma configuration file.
```

We should also change the example in the README where `localhost` is required in `config.ru`.